addons/product/models/product_pricelist.py

```diff
@@ -531,3 +507,30 @@ def write(self, values):
         # to be invalided and recomputed.
         self.invalidate_cache()
         return res
+
+    def _compute_price(self, price, price_uom, product, quantity):
```

Comment:
I'd give default values for unused parameters:
```suggestion
    def _compute_price(self, price, price_uom, product, quantity=1.0, partner=False):
```

By the way a docstring could be helpful to understand why we define unused parameters. Otherwise these might be removed in a future refactoring.